src/TableDownloadButton/TableDownloadButton.styles.ts

```diff
@@ -2,15 +2,32 @@ import styled, { css } from 'styled-components';
 
 interface Props {
   readonly variant: string;
+  disabled?: boolean;
+  children?: React.ReactNode;
 }
 
 export const TableActionButton = styled.button<Props>`
   border-radius: 0px 0px 0px 2px;
```

Comment:
- [ ] Can you remove the border-radius?
- [ ] Can you make the border `2px solid ${p.theme.colors.skyBlue}`?